src/color-picker/color-picker.scss

```diff
@@ -12,6 +12,9 @@ $iui-active-box-shadow: 0 0 0 $iui-xxs rgba(t(iui-color-foreground-primary-rgb),
 $iui-hover-box-shadow: 0 0 1px $iui-xxs + 1 rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5));
 
 @mixin iui-color-picker {
+  --iui-color-field-hue: initial;
+  --iui-color-picker-selected-color: initial;
```

Comment:
And here is interesting situation... `selected-color` is not even used in this mixin, its used in others, so where to define it?
`hue` same, but its used in only 1 mixin, so it should go there

Reply:
It is used in multiple places so I moved it up to parent

Reply:
Hue is not, can be moved under mixin where its used

Reply:
Ok I see. If you look at the HTML (or JSX), the variable is passed in the color picker root element, so setting it to `initial` inside a child would override that value.

Reply:
so move `--iui-color-picker-hue` under `iui-color-swatch` and lets complete this pr